js/docEnums.js

```diff
@@ -70,6 +70,16 @@
  * @member 'Default' @deprecated 'Material'
  */
 
+/**
+ * @typedef {string} Enums.VizWordWrap
+ * @enum {'normal'|'break-word'|'none'}
+ */
+
+/**
+  * @typedef {string} Enums.VizOverflow
```

Comment:
I suggest to rename to 'VizTextOverflow'

Reply:
@RomanTsukanov ?

Reply:
VizTextOverflow is good